### PR TITLE
Dbsetup

### PR DESCRIPTION
**Pull request by FrancoDominguez**
Created the Mysqlcon.java file to write sql queries to the db, I left an example in the Driver.java class. The db is active and working, you will only need the .env file. Wrote extra instructions in the readme. I setup the directory structure for java and its dependencies.

### CODE REVIEW

File: backend/pom.xml [hunk 1]
```diff
@@ -0,0 +1,59 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>SOEN-342</groupId>
```
Comment (MarkGhaby) [inline backend/pom.xml]: soen 343****

Comment (FrancoDominguez) [reply backend/pom.xml]: yes thank you

**Comment by MarkGhaby** [comment]
Approved!!!!!!!!!!

**Comment by antoinemansour7** [comment]
g2go

**Pull request merged**